qa/full-cluster-restart/src/test/java/org/elasticsearch/upgrades/FullClusterRestartIT.java

```diff
@@ -1278,7 +1278,7 @@ public void testOperationBasedRecovery() throws Exception {
                 }
             }
             flush(index, true);
-            ensurePeerRecoveryRetentionLeasesRenewedAndSynced(index);
+            ensurePeerRecoveryRetentionLeasesRenewedAndSynced(index, false);
```

Comment:
Should we set `alwaysExists ` to `true` if `minimumNodeVersion()` is on or after 7.6 (after backport) (here as well as in other places)?

Reply:
Sure, will do.